resume/resume.html

```diff
@@ -30,9 +30,8 @@
         </div> -->
     </header>
 
-    <div class="body-container">
+    <main>
```

Comment:
Same comment about leaving the class name on the 'main' element so we can style it the same. Then we can leave the css file as it was!